src/client/features/view/components/menu/index.js

```diff
@@ -51,6 +52,10 @@ class Menu extends React.Component {
       }
     });
     layout.run();
+    layout.pon('layoutstop').then(function( event ){
```

Comment:
I think this can be combined with the other layoutstop function 8 lines up.

Reply:
It indeed can. The other one was just recently added by another PR. I have combined the two together